mplc/mpl_utils.py

```diff
@@ -132,7 +132,7 @@ def aggregate_model_weights(self):
             avg_weights_for_layer = np.average(
                 np.array(weights_for_layer), axis=0, weights=self.aggregation_weights
             )
-            new_weights.append(list(avg_weights_for_layer))
+            new_weights.append(avg_weights_for_layer)
```

Comment:
Is that really an innocent change?

Reply:
Tf wants a list of np.array now, so we need it